.github/release_workflow.md

```diff
@@ -77,3 +77,5 @@ Some other essential things to check throughout the release process -
   git tag -f <tag_name> <commit_hash>
   git push -f <pybamm-team/PyBaMM_remote_name> <tag_name>  # can only be carried out by the maintainers
   ```
+- If changes are made to the API, console scripts, entry points, new optional dependencies are added, support for major Python versions is dropped or added, or core project information and metadata are modified at the time of the release, make sure to update the `meta.yaml` file in the `recipe/` folder of the [conda-forge/pybamm-feedstock](https://github.com/conda-forge/pybamm-feedstock) repository accordingly by following the instructions in the [conda-forge documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-feedstock-repository) and re-rendering the recipe
+- The conda-forge release workflow will automatically be triggered following a stable PyPI release, and the aforementioned updates can be carried out either in a personal fork of the feedstock repository, or directly in the main repository by pushing changes to the automated PR created by the conda-forge-bot.
```

Comment:
```suggestion
- The conda-forge release workflow will automatically be triggered following a stable PyPI release, and the aforementioned updates should be carried out directly in the main repository by pushing changes to the automated PR created by the conda-forge-bot. A manual PR can also be created if the updates are not included in the automated PR for some reason. This manual PR must bump the build number in `meta.yaml`.
```

Reply:
This information is given in the link to the conda-forge documentation

Reply:
Adding it as per your suggestion because it would help as a failsafe